src/sentry/notifications/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from sentry.notifications.models.notificationsettingprovider import NotificationSettingProvider
+
+__all__ = ("NotificationSettingProvider",)
```

Comment:
You'll likely get conflicts with the barrel file.

Reply:
😞